pkg/api/v1alpha1/provider_spec.go

```diff
@@ -51,6 +51,10 @@ type GCPProviderSpec struct {
 	// must be created before you can assign them.
 	Disks []*GCPDisk `json:"disks,omitempty"`
 
+	// GPUConfig: Configurations related to GPU which would be attached to the instance. Enough
+	// Quota of the particular GPU should be available.
+	GPUConfig *GPUConfig `json:"gpuConfig,omitempty"`
```

Comment:
Can this be renamed to just `Gpu` just like `Disks`. Also the type is `GCP` specific and going by the current convention where you have a `GCP` prefix to Metadata, Disk, Scheduling, ServiceAccount the type should then be called `GCPGpu`

Reply:
sure, will do that.